aiohttp/parsers.py

```diff
@@ -224,6 +225,22 @@ def __init__(self, transport, protocol, reader, loop):
         self._protocol = protocol
         self._reader = reader
         self._loop = loop
+        self._tcp_nodelay = False
+        self._socket = transport.get_extra_info('socket')
+
+    @property
+    def tcp_nodelay(self):
+        return self._tcp_nodelay
+
+    def set_tcp_nodelay(self, value):
```

Comment:
I though about property's setter but found that syscall worth explicit function call.